libandroid-navigation/build.gradle

```diff
@@ -49,6 +49,8 @@ dependencies {
   // Logging
   implementation dependenciesList.timber
 
+  implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
Wondering if this is needed as part of this PR 🤔
I guess this slipped in by mistake.